tests/misc_env/install_prereq.py

```diff
@@ -147,17 +147,17 @@ def setup_subscription_manager(ceph, is_production=False, timeout=1800):
             command = "sudo subscription-manager --force register "
             if is_production:
                 command += "--serverurl=subscription.rhsm.redhat.com:443/subscription "
-
                 username_ = config_["cdn_credentials"]["username"]
                 password_ = config_["cdn_credentials"]["password"]
+                pool_id = "8a85f98960dbf6510160df23eb447470"
```

Comment:
can we pass pool Id from location where we fetch creds?

Reply:
i would suggest the same, but there are places where the pool ids are hardcoded in the code. we can plan to unify all the pool ids into a file and take it from there. I think that can be separate PR. 
but nonetheless.